test/testpermissions.cpp

```diff
@@ -111,18 +116,58 @@ private slots:
         assertCsyncJournalOk(fakeFolder.syncJournal());
         qInfo("Do some changes and see how they propagate");
 
+        const auto removeReadOnly = [&] (const QString &file)  {
+            auto fileInfoToDelete = QFileInfo(fakeFolder.localPath() + file);
+            QFile(fakeFolder.localPath() + file).setPermissions(QFile::WriteOwner | QFile::ReadOwner);
+            const auto readOnlyFolder = !static_cast<bool>(std::filesystem::status(fileInfoToDelete.absolutePath().toStdWString()).permissions() & std::filesystem::perms::owner_write);
```

Comment:
@mgallien same question, why readOnly`Folder` for an absolute path of a `file`?

Reply:
changed to `isReadOnly`